timelapse.py

```diff
@@ -67,36 +67,46 @@ def capture_image():
         set_camera_options(camera)
 
         # Capture a picture.
-        camera.capture(dir + '/image{0:05d}.jpg'.format(image_number))
+        camera.capture(dir + '/image{0:08d}.jpg'.format(image_number))
+        sleep(1)
```

Comment:
What's the purpose of adding a 1 second sleep here? If you have shorter intervals, this could interfere with the timing.